src/pgduckdb_ddl.cpp

```diff
@@ -160,15 +158,7 @@ duckdb_create_table_trigger(PG_FUNCTION_ARGS) {
 		elog(ERROR, "Unexpected parsetree type: %d", nodeTag(parsetree));
 	}
 
-	std::string query_string(pgduckdb_get_tabledef(relid));
-
-	auto connection = pgduckdb::DuckDBManager::Get().GetConnection();
-	auto &context = *connection->context;
-	auto result = context.Query(query_string, false);
-	if (result->HasError()) {
-		elog(ERROR, "(PGDuckDB/duckdb_create_table_trigger) Could not create table: %s", result->GetError().c_str());
-	}
-
+	pgduckdb::DuckDBQueryOrThrow(pgduckdb_get_tabledef(relid));
```

Comment:
It seems incorrect to go from `elog` to `throw` here. This will cause a Postgres crash currently.

Reply:
Ah yes - I extracted it from my PR that wraps all functions with try/catch but didn't pay attention here. Will review other places too.

Reply:
I've made these function "throw" PG errors.